src/readii/loaders.py

```diff
@@ -49,11 +54,12 @@ def loadRTSTRUCTSITK(
         A dictionary of mask ROIs from the loaded RTSTRUCT image as a SimpleITK image objects.
         The segmentation label is set to 1.
     """
-
     # Set up segmentation loader
+    logger.debug(f"Making mask using ROI names: {roiNames}")
     makeMask = StructureSetToSegmentation(roi_names=roiNames)
```

Comment:
in 
```
def loadRTSTRUCTSITK(
    rtstructPath: str, baseImageDirPath: str, roiNames: Optional[str] = None
) -> dict:
```

roiNames can be `None`

but `StructureSetToSegmentation` expects it to be a `Dict[str, str]`

Reply:
From the docstring in the MIT code:
```If `roi_names` is kept empty ([]), the pipeline will process all ROIs/contours found according to their original names.```
[https://github.com/bhklab/med-imagetools/blob/5546cec94f1cfe1e3fb4a2dff28a4c1e5275216e/src/imgtools/ops/ops.py#L1467](url)

So I think that's something that can be corrected in the MIT implementation.

Reply:
ah yes, looks like the type annotations in MIT needs to be updated to reflect that for roi_names 

```
def __init__(self, 
                 roi_names: Dict[str, str], 
                 continuous: bool = True):
        """Initialize the op.

        Parameters
        ----------
        roi_names
            List of ROI names to export. Both full names and
            case-insensitive regular expressions are allowed.
            All labels within one sublist will be assigned
            the same label.
```
`Dict` threw me off tbh 

@skim2257